app/helpers/is-empty.js

```diff
@@ -0,0 +1,8 @@
+import { isEmpty } from '@ember/utils';
```

Comment:
ember-truth-helpers already has an `is-empty` helper. Can we use that one instead?